README.md

```diff
@@ -2,60 +2,106 @@
-| [`with-solana-passkeys`](/examples/with-solana-passkeys/)              | A NextJS app powering users to create suborgs, sign messages, and create transactions sponsored by the parent org using @turnkey/solana |
-| [`with-eth-passkey-signers`](/examples/with-eth-passkey-signers/)      | A NextJS app powering users to create suborgs and sign messages via Viem or Ethers                                                      |
-| [`with-eip-1193-provider`](/examples/with-eip-1193-provider/)          | A NextJS app that demonstrates how to use Turnkey the `@turnkey/eip-1193-provider` in your app                                          |
-| [`with-wallet-stamper`](/examples/with-wallet-stamper/)                | A NextJS app that demonstrates how to use Turnkey the `@turnkey/wallet-stamper` in your app                                             |
+### Best First Examples -- Test in Stackblitz
+
+The following code examples have been loaded into Stackblitz web environments so you can test them out immediately
+
+| Example                                                           | Description                                                                                                                             | Stackblitz Link                                                                      |
+| ----------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------ |
+| [`email-auth-local-storage`](/examples/email-auth-local-storage/) | A NextJS app demonstrating a complete email auth flow using a locally stored target embedded key                                        | [Test it out on Stackblitz!](https://stackblitz.com/edit/stackblitz-starters-pyyw59) |
+| [`with-eth-passkey-signers`](/examples/with-eth-passkey-signers/) | A NextJS app powering users to create suborgs and sign messages via Viem or Ethers                                                      | [Test it out on Stackblitz!](https://stackblitz.com/edit/stackblitz-starters-2psu3g) |
+| [`with-solana`](/examples/with-solana/)                           | Create a new Solana address, then sign and broadcast a transaction on Solana's devnet                                                   | [Test it out on Stackblitz!](https://stackblitz.com/edit/stackblitz-starters-xeb93i) |
+| [`with-solana-passkeys`](/examples/with-solana-passkeys/)         | A NextJS app powering users to create suborgs, sign messages, and create transactions sponsored by the parent org using @turnkey/solana | [Test it out on Stackblitz!](https://stackblitz.com/edit/stackblitz-starters-h5pmnu) |
+
+### Other Code Examples
+
```

Comment:
(optional) might benefit from a word or two about the fact that these examples require a local installation of node + following instruction in linked READMEs? (since you have an intro in the stackblitz examples section)